web/templates/dropdown_list.hbs

```diff
@@ -1,18 +1,29 @@
 {{#with item}}
     <li class="list-item" role="presentation" data-unique-id="{{unique_id}}" data-name="{{name}}" tabindex="0">
         {{#if description}}
-        <a class="dropdown-list-item-common-styles">
+        <span class="dropdown-list-item-common-styles">
             <span class="dropdown-list-item-name">
                 {{#if bold_current_selection}}
                     <span class="dropdown-list-bold-selected">{{name}}</span>
+                    {{#if is_saved_reply}}
+                        <span id="delete-saved-reply-id" class="delete-saved-reply">
+                            <i class="fa fa-trash-o delete-saved-reply"></i>
+                        </span>
+                    {{/if}}
```

Comment:
A few notes on this:
- Don't use the same `id` for an element that will have multiple copies of it exist; it's invalid HTML to have duplicate IDs. You also have the same class on both the wrapping `span` element and the inner `i` element. `delete-saved-reply-icon` for the inner one and `delete-saved-reply` on the outer one would be a fine class naming scheme.
- That all said, we also want to have this file be as generic as possible, not filled with customizations for specific features. I think you could just rename `is_saved_reply` to `has_delete_icon`, call the classes `dropdown-list-delete` / `dropdown-list-delete-icon`, and achieve that goal with pretty minimal effort.